packages/durable-objects/src/accounts-do.ts

```diff
@@ -0,0 +1,88 @@
+import { DurableObjectState } from '@cloudflare/workers-types';
+import { APIEnv, SentinalRequest } from './types';
+import { unique } from '@notional-finance/helpers';
+
+export class AccountsDO {
+  state: DurableObjectState;
+  env: APIEnv;
+
+  constructor(state: DurableObjectState, env: APIEnv) {
+    this.state = state;
+    this.env = env;
+  }
+
+  async fetch(request: Request): Promise<Response> {
+    try {
+      const req = request.clone();
+      const { method } = request;
+      if (method !== 'POST' && method !== 'GET') {
+        return new Response('Method Not Allowed', { status: 405 });
+      }
+      if (method === 'POST') {
+        return this.update(req);
+      } else if (method === 'GET') {
+        return this.get(req);
+      }
+      return new Response('Not Found', { status: 404 });
+    } catch (error) {
+      console.log(error);
+      return new Response((error as Error).message, { status: 400 });
+    }
+  }
+
+  async update(request: Request) {
+    try {
+      const { events }: SentinalRequest = await request.json();
+      if (events && events.length > 0) {
+        const network = events[0].sentinel.network;
+        const accounts = unique(events.map((e) => e.transaction.from));
```

Comment:
I don't think this is the right address, the SentinalRequest should have a variable called `account` on it. The address being stored here is the transaction sender (which may not be the actual address we care about).

Reply:
That's what the existing auto task was using, it recorded my wallet addresses.  I can double check there isn't another field we should use